### ISSUE TITLE
Restart inputs are different from past inputs

### ISSUE THREAD

**Issue by YingYa** [griffithlab/pVACtools#1076]
### Installation Type

Docker

### pVACtools Version / Docker Image

4.1.0 - running via Singularity (v3.8.1)

### Operating System

CentOS7

### Describe the bug

**Running pVACtools using Singularity on a CentOS Linux 7 HPC at work. (Docker not permitted)**

Re-running the interrupted shell will be aborted.

**The key information from the stdout:**
......
Executing MHC Class II predictions

**The error information from stderr:**
Restart inputs are different from past inputs:
Past input: alleles - ['DPB1*04:01', 'DPA1*01:03', 'DRB1*13:03', 'DQB1*04:02', 'DQA1*04:01-DQB1*03:01', 'DRB1*08:01', 'DQA1*04:01', 'DPA1*01:03-DPB1*04:01', 'DQA1*04:01-DQB1*04:02', 'DQB1*03:01']
Current input: alleles - ['DPB1*04:01', 'DQA1*04:01', 'DQA1*04:01-DQB1*03:01', 'DRB1*13:03', 'DPA1*01:03-DPB1*04:01', 'DQA1*04:01-DQB1*04:02', 'DPA1*01:03', 'DQB1*03:01', 'DRB1*08:01', 'DQB1*04:02']
Aborting.

### How to reproduce this bug

```Shell
singularity exec -e -B /path/to/indir:/indir -B `pwd` /path/to/pvactools-4.1.0.sif pvacseq run /indir/test3.vcf.gz test3 HLA-A*02:01,HLA-B*27:05,HLA-B*41:02,HLA-C*01:02,HLA-C*17:01,DPA1*01:03,DPB1*04:01,DQA1*04:01,DQB1*03:01,DQB1*04:02,DRB1*08:01,DRB1*13:03 all . --iedb-install-directory /opt/iedb -t 4 --minimum-fold-change 1 --tdna-vaf 0.1 --trna-vaf 0.1 -a sample_name --normal-sample-name normal3 --tumor-purity 0.4 -p /indir/test3.phased.vep.vcf.gz
```

### Log output

Restart inputs are different from past inputs:
Past input: alleles - ['DPB1*04:01', 'DPA1*01:03', 'DRB1*13:03', 'DQB1*04:02', 'DQA1*04:01-DQB1*03:01', 'DRB1*08:01', 'DQA1*04:01', 'DPA1*01:03-DPB1*04:01', 'DQA1*04:01-DQB1*04:02', 'DQB1*03:01']
Current input: alleles - ['DPB1*04:01', 'DQA1*04:01', 'DQA1*04:01-DQB1*03:01', 'DRB1*13:03', 'DPA1*01:03-DPB1*04:01', 'DQA1*04:01-DQB1*04:02', 'DPA1*01:03', 'DQB1*03:01', 'DRB1*08:01', 'DQB1*04:02']
Aborting.

**Comment by susannasiebert** [comment]
Thank you for this bug report. I have a PR out to fix this issue and it will go out with the next release. In the meantime you can circumvent this issue by deleting the inputs.yml file in the log subdirectory of your MHC_Class_II output directory.

**Comment by YingYa** [comment]
Thanks for the suggestions. I will remove the inputs.yml file to rerun the shell.

**Comment by susannasiebert** [comment]
This issue has been fixed in version 4.1.1. Please give that version a try and feel free to reopen this issue if the error persists.

**Comment by toprakdog** [comment]
Hi! Thank you for creating this great work. Regarding the issue, are you sure it's fixed? I'm still experiencing a similar error with version 4.1.1.
The error message is;
pvacseq run \
 /pvacseq_example_data/1007-aFT.wgs.hg38.soma.hard-filtered.vcf_output.vcf \
1007-aFT-wgs \
 HLA-A*29:02,HLA-B*45:01,DRB1*04:05 \
 all \
/pvacseq_output_data \
-e1 8,9,10 \
 -e2 15 \
--normal-sample-name 1007-PBcfA-wgs \
--iedb-install-directory /opt/iedb
Executing MHC Class I predictions
Restart to be executed with a different pVACtools version:
Past version: 4.0.8
Current version: 4.1.1
Restart inputs are different from past inputs:

There might be something wrong with my code too. I am using Docker and mounted directories as suggested.
Thank you for your time.

**Comment by susannasiebert** [comment]
Yes, that is to be expected because your rerun was attempted with a different pVACtools version. This particular error from the original ticket was in regards to logged class II alleles specifically, which should now be fixed.

For your situation, you may still try to delete the log/inputs.yml file but we can generally make no guarantee that that won't lead to other errors since there might be differences in the temporary file formats between major/minor pVACtools versions like moving from 4.0 to 4.1. In this particular case I don't believe that that should cause any problems though.
